packages/anasazi/src/AnasaziBlockKrylovSchurSolMgr.hpp

```diff
@@ -388,7 +389,7 @@ BlockKrylovSchurSolMgr<ScalarType,MV,OP>::BlockKrylovSchurSolMgr(
 
   // which orthogonalization to use
   ortho_ = pl.get("Orthogonalization",ortho_);
-  if (ortho_ != "DGKS" && ortho_ != "SVQB") {
+  if (ortho_ != "DGKS" && ortho_ != "SVQB" && ortho_ != "ICGS") {
```

Comment:
Anasazi doesn't have "IMGS" (while Belos does), right?

Reply:
Correct.  There isn't IMGS in Anasazi, only those three methods.  An orthogonalization factory would be preferable, but was never implemented in Anasazi.

Reply:
Cool; thanks for explaining!